src/aws_encryption_sdk/materials_managers/mpl/cmm.py

```diff
@@ -0,0 +1,142 @@
+"""Retrieves encryption/decryption materials from the MPL."""
+
+# These dependencies are only loaded if you install the MPL.
+try:
+    # pylint seems to struggle with this conditional import
+    # pylint: disable=unused-import
+    from aws_cryptographic_materialproviders.mpl.errors import AwsCryptographicMaterialProvidersException
+    from aws_cryptographic_materialproviders.mpl.models import (
+        AlgorithmSuiteIdESDK,
+        CommitmentPolicyESDK,
+        DecryptMaterialsInput,
+        DecryptMaterialsOutput,
+        EncryptedDataKey as MPL_EncryptedDataKey,
+        GetEncryptionMaterialsInput,
+        GetEncryptionMaterialsOutput,
+    )
+    from aws_cryptographic_materialproviders.mpl.references import ICryptographicMaterialsManager
```

Comment:
Issue/Suggestion: I think I would preface all of these with `MPL_`.

The names are so similar, I wanted to go the distance to avoid confusion.